client/packages/dashboard/src/api/operations.graphql

```diff
@@ -32,6 +32,9 @@ query requisitionCounts($storeId: String!) {
     response {
       new
     }
+    emergency {
+      new
+    }
```

Comment:
Have added emergency query into other requisition count queries.

This seemed a lot more simple than adding it to its own hook - but maybe I'm not aware of a use case where it would need its own hook?